tests/testthat/test_pipeop_learnerquantiles.R

```diff
@@ -0,0 +1,182 @@
+  expect_string(po$id)
+})
+
+test_that("PipeOpLearnerQuantiles - model active binding to state", {
+  lrn = mlr_learners$get("regr.debug")
+  po = PipeOpLearnerQuantiles$new(lrn)
+  task = mlr_tasks$get("mtcars")
+
+  # before training states are NULL
+  expect_null(po$state)
+  expect_null(po$learner$state, po$state)
+  expect_equal(po$learner, po$learner_model)
+  expect_null(po$learner_model$state, po$state)
+
+  train_out = po$train(list(task))
+  train_state = po$state
+
+  # after predicting states are unchanged
+  predict_out = po$predict(list(task))
+  expect_equal(po$state, train_state)
+})
+
+test_that("PipeOpLearnerQuantiles - predict_type is fixed", {
+  lrn = lrn("regr.debug")
+  po = PipeOpLearnerQuantiles$new(lrn)
+  expect_equal(po$predict_type, c("response", "quantiles"))
+})
+
+test_that("PipeOpLearnerQuantiles - integration with larger graph", {
+  task = mlr_tasks$get("mtcars")
+  lrn = mlr_learners$get("regr.debug")
+
+  po_quantiles = PipeOpLearnerQuantiles$new(lrn)
+  po_nop = PipeOpNOP$new()
+  graph = po_quantiles %>>% po_nop
+
+  graph$train(task)
+  predictions = graph$predict(task)[[1]]
+
+  expect_matrix(predictions$quantiles)
+  expect_true("response" %in% names(predictions))
+  expect_true(inherits(predictions, "PredictionRegr"))
+})
+
+test_that("marshal", {
+  lrn = lrn("regr.debug")
+  lrn$properties = c(lrn$properties, "marshal")
+
+  task = tsk("mtcars")
+  po = PipeOpLearnerQuantiles$new(lrn)
+  po$train(list(task))
+  po_state = po$state
+
+  expect_class(po_state, "pipeop_learner_quantiles_state")
+
+  po_state_marshaled = marshal_model(po_state, inplace = FALSE)
+
+  expect_class(po_state_marshaled, "pipeop_learner_quantiles_state_marshaled")
+  expect_true(is_marshaled_model(po_state_marshaled))
+
+  po_state_unmarshaled = unmarshal_model(po_state_marshaled)
+  expect_equal(po_state, po_state_unmarshaled)
+})
+
+test_that("marshal multiplicity", {
+  lrn = lrn("regr.debug")
+  lrn$properties = c(lrn$properties, "marshal")
+  po = PipeOpLearnerQuantiles$new(lrn)
+
+  task1 = mlr_tasks$get("mtcars")
+  task2 = mlr_tasks$get("boston_housing")
+
+  po$train(list(Multiplicity(task1, task2)))
+  state = po$state
+
+  marshaled_state = marshal_model(state)
+  expect_class(po$state, "Multiplicity")
+
+  unmarshaled_state = unmarshal_model(marshaled_state)
+  expect_equal(state, unmarshaled_state)
+})
+
+test_that("state class and multiplicity", {
+  lrn = lrn("regr.debug")
+  lrn$properties = c(lrn$properties, "marshal")
+  po = PipeOpLearnerQuantiles$new(lrn)
+  po$train(list(Multiplicity(tsk("mtcars"))))
+  expect_class(po$state, "Multiplicity")
+  expect_class(po$state[[1L]], "pipeop_learner_quantiles_state")
+
+  # recursive
+  po1 = po("learner_quantiles", learner = lrn("regr.debug"))
+  po1$train(list(Multiplicity(Multiplicity(tsk("mtcars")))))
+  expect_class(po1$state, "Multiplicity")
+  expect_class(po1$state[[1L]], "Multiplicity")
+  expect_class(po1$state[[1L]][[1L]], "pipeop_learner_quantiles_state")
```

Comment:
can you also test that `$learner_model` works correctly for multiplicities?

Reply:
It should contain a multiplicity with an entry for each input

Reply:
also check that predictions work with multiplicities

Reply:
Yes, I added tests for multiple inputs, `$learner_model`, and prediction with multiplicities.